lib/Runtime/Library/TypedArray.cpp

```diff
@@ -175,6 +171,11 @@ namespace Js
                             Var lengthVar = JavascriptOperators::OP_GetProperty(jsArraySource, PropertyIds::length, scriptContext);
                             if (JavascriptOperators::GetTypeId(lengthVar) == TypeIds_Undefined)
```

Comment:
If there is a length property defined on the iterable object, is the new semantics right?